internal/client/client_test.go

```diff
@@ -74,103 +58,57 @@ func TestAPIResponse(t *testing.T) {
-	if err != nil {
-		t.Fatalf("GetRaw failed with a client error: %v", err)
-	}
-	if resp != nil {
-		t.Fatalf("GetRaw request failed: %v", err)
-	}
-	if len(b) == 0 {
-		t.Fatal("GetRaw returned an empty string")
-	}
+	require.NoError(t, err)
+	require.Nil(t, resp)
+	require.Greater(t, len(b), 0)
+
 	// Get an API error
 	b, resp, err = GetRaw(testState.socket, "GET", "/api/v0/blueprints/list?offset=bad")
-	if err != nil {
-		t.Fatalf("GetRaw bad request failed with a client error: %v", err)
-	}
-	if resp == nil {
-		t.Fatalf("GetRaw bad request did not return an error: %v", b)
-	}
-	if resp.Status != false {
-		t.Fatalf("Status != false: %#v", resp)
-	}
-	if len(resp.AllErrors()) < 1 {
-		t.Fatalf("GetRaw error did not return error message: %#v", resp)
-	} else if resp.Errors[0].ID != "BadLimitOrOffset" {
-		t.Fatalf("GetRaw error ID is not BadLimitOrOffset: %#v", resp)
-	}
+	require.NoError(t, err)
+	require.NotNil(t, resp)
+	require.False(t, resp.Status)
+	require.GreaterOrEqual(t, len(resp.AllErrors()), 1)
+	require.Equal(t, "BadLimitOrOffset", resp.Errors[0].ID)
 }
 
 func TestGetJSONAll(t *testing.T) {
 	// Get all the projects
 	b, resp, err := GetJSONAll(testState.socket, "/api/v0/projects/list")
-	if err != nil {
-		t.Fatalf("GetJSONAll failed with a client error: %v", err)
-	}
-	if resp != nil {
-		t.Fatalf("GetJSONAll request failed: %v", resp)
-	}
-	if len(b) < 100 {
-		t.Fatalf("GetJSONAll response is too short: %#v", b)
-	}
+	require.NoError(t, err)
+	require.Nil(t, resp)
+	require.GreaterOrEqual(t, len(b), 100)
 
 	// Run it on a route that doesn't support offset/limit
 	b, resp, err = GetJSONAll(testState.socket, "/api/status")
-	if err == nil {
-		t.Fatalf("GetJSONAll bad route failed: %v", b)
-	}
-	if err.Error() != "Response is missing the total value" {
-		t.Fatalf("GetJSONAll bad route has unexpected total value: %v", resp)
-	}
+	require.EqualError(t, err, "Response is missing the total value")
 }
 
 func TestPostRaw(t *testing.T) {
 	// There are no routes that accept raw POST w/o Content-Type so this ends up testing the error path
-	b, resp, err := PostRaw(testState.socket, "/api/v0/blueprints/new", "nobody", nil)
-	if err != nil {
-		t.Fatalf("PostRaw bad request failed with a client error: %v", err)
-	}
-	if resp == nil {
-		t.Fatalf("PostRaw bad request did not return an error: %v", b)
-	}
-	if resp.Status != false {
-		t.Fatalf("PostRaw bad request status != false: %#v", resp)
-	}
-	if len(resp.AllErrors()) < 1 {
-		t.Fatalf("GetRaw error did not return error message: %#v", resp)
-	} else if resp.Errors[0].ID != "BlueprintsError" {
-		t.Fatalf("GetRaw error ID is not BlueprintsError: %#v", resp)
-	}
+	_, resp, err := PostRaw(testState.socket, "/api/v0/blueprints/new", "nobody", nil)
+	require.NoError(t, err)
+	require.NotNil(t, resp)
+	require.False(t, resp.Status)
+	require.GreaterOrEqual(t, len(resp.AllErrors()), 1)
+	require.Equal(t, "BlueprintsError", resp.Errors[0].ID)
```

Comment:
Add error messages to print the value of resp to both of these.

Reply:
added